elf_reader.go

```diff
@@ -344,6 +344,19 @@ func (ec *elfCode) loadInstructions(section *elfSection) (asm.Instructions, uint
 	}
 }
 
+func offsetToSymbol(section *elfSection, offset int64) (elf.Symbol, error) {
+	if offset < 0 {
+		return elf.Symbol{}, fmt.Errorf("invalid offset %d", offset)
```

Comment:
This isn't much more informative than `no symbol at offset -1` from the error below, so I'd probably skip it. Which means this function is necessary.